src/main/thrift/parquet.thrift

```diff
@@ -932,27 +942,30 @@ struct FileMetaData {
   7: optional list<ColumnOrder> column_orders
   
   /** 
-   * Encryption algorithm. Note that this field is only used for files
-   * with plaintext footer. Files with encrypted footer store the algorithm id
+   * Encryption algorithm. This field is set only in encrypted files
+   * with plaintext footer. Files with encrypted footer store algorithm id
    * in FileCryptoMetaData structure.
    */
   8: optional EncryptionAlgorithm encryption_algorithm
+
+  /** 
+   * Retrieval metadata of key used for signing the footer. 
+   * Used only in encrypted files with plaintext footer. 
+   */ 
+  9: optional binary footer_signing_key_metadata
 }
 
 /** Crypto metadata for files with encrypted footer **/
 struct FileCryptoMetaData {
   /** 
-   * Encryption algorithm. Note that this field is only used for files
-   * with encrypted footer. Files with plaintext footer store the algorithm id
+   * Encryption algorithm. This field is only used for files
+   * with encrypted footer. Files with plaintext footer store algorithm id
    * inside footer (FileMetaData structure).
    */
   1: required EncryptionAlgorithm encryption_algorithm
     
   /** Retrieval metadata of key used for encryption of footer, 
    *  and (possibly) columns **/
-  2: optional binary footer_key_metadata
-
-  /** Offset of encrypted Parquet footer **/
-  3: required i64 footer_offset
```

Comment:
The 'footer_offset' field still exists in the diagram at the section 'Encrypted footer mode' of the design document (https://github.com/ggershinsky/parquet-format/blob/p1232-encryption-docs/Encryption.md).

Reply:
this is an outdated link. the current doc is at
https://github.com/apache/parquet-format/blob/encryption/Encryption.md

Reply:
Please update the Parquet-1178(https://docs.google.com/document/d/1T89G7xR0zHFV1f2pjTO28jtfVm8qoNVGEJQ70Rsk-bY) to let it point to the current doc.

Reply:
Sure, will do.